mindsdb/integrations/handlers/lindorm_handler/lindorm_handler.py

```diff
@@ -0,0 +1,266 @@
+                    data_frame=pd.DataFrame(
+                        result,
+                        columns=[x[0] for x in cursor.description]
+                    )
+                )
+            else:
+                connection.commit()
+                response = Response(RESPONSE_TYPE.OK)
+        except Exception as e:
+            log.logger.error(f'Error running query: {query} on the Lindorm Query Server!')
+            response = Response(
+                RESPONSE_TYPE.ERROR,
+                error_message=str(e)
+            )
+
+        cursor.close()
+        if need_to_close is True:
+            self.disconnect()
+
+        return response
+
+    def query(self, query: ASTNode) -> StatusResponse:
+        """
+        Receive query as AST (abstract syntax tree) and act upon it somehow.
+        Args:
+            query (ASTNode): sql query represented as AST. May be any kind
+                of query: SELECT, INTSERT, DELETE, etc
+        Returns:
+            HandlerResponse
+        """
+
+        renderer = SqlalchemyRender(PhoenixDialect)
+        query_str = renderer.get_string(query, with_failback=True)
+        return self.native_query(query_str)
+
+    def get_tables(self) -> StatusResponse:
+        """
+        Return list of entities that will be accessible as tables.
+        Returns:
+            HandlerResponse
+        """
+
+        query = """
+            SELECT DISTINCT TABLE_NAME, TABLE_SCHEM FROM SYSTEM.CATALOG
+        """
+        result = self.native_query(query)
+        df = result.data_frame
+        df = df[df['TABLE_SCHEM'] != 'SYSTEM']
+        df = df.drop('TABLE_SCHEM', axis=1)
+        result.data_frame = df.rename(columns={df.columns[0]: 'table_name'})
+        return result
+
+    def get_columns(self, table_name: str) -> StatusResponse:
+        """
+        Returns a list of entity columns.
+        Args:
+            table_name (str): name of one of tables returned by self.get_tables()
+        Returns:
+            HandlerResponse
+        """
+
+        need_to_close = self.is_connected is False
+
+        connection = self.connect()
+        cursor = connection.cursor()
+
+        try:
+            query = f"SELECT * from {table_name} LIMIT 5"
+            cursor.execute(query)
+            cursor.fetchall()
+
+            response = Response(
+                RESPONSE_TYPE.TABLE,
+                data_frame=pd.DataFrame(
+                    [(x[0], x[1]) for x in cursor.description],
+                    columns=['column_name', 'data_type']
+                )
+            )
+
+        except Exception as e:
+            log.logger.error(f'Error running query: {query} on the Phoenix Query Server!')
+            response = Response(
+                RESPONSE_TYPE.ERROR,
+                error_message=str(e)
+            )
+
+        cursor.close()
+        if need_to_close is True:
+            self.disconnect()
+
+        return response
+
+
+connection_args = OrderedDict(
```

Comment:
Let's add all of the bellow keys to the README

Reply:
added the inputs into README in latest commit